lib/lookup.json

```diff
@@ -88,7 +88,8 @@
     "prefix": "v"
   },
   "coffeescript": {
-    "maintainers": ["jashkenas", "GeoffreyBooth"]
+    "maintainers": ["jashkenas", "GeoffreyBooth"],
+    "skip": "rhel"
```

Comment:
I know `rhel` is used elsewhere in this file, but really we mean `s390` (it just happens that the only nodes in the CI with rhel in them are Linux ONE).

Reply:
will an s390 check work

Reply:
s390 is used in many places in the file